yamf/models/ErrorMessage.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-require_once 'View.php';
+namespace Yamf\models;
```

Comment:
This works based on how I did the files, but in looking at it, `models` should probably be capitalized. Let's have the namespace for YAMF models be `Yamf\Models` instead. In a quick look at the [PSR-4 Spec](https://www.php-fig.org/psr/psr-4/) it looks like they use capitals there. So, please update all the `Yamf\models` to be `Yamf\Models`.

Reply:
Hmmm........I'm torn because matching the namespace capitalization would be nice, but capital letters are annoying in file paths when typing in the console. 😋 The PSR-4 spec does not specify, and the examples go both ways.

I think I will go with not changing the actual folder names at this point. I personally prefer lowercase folder names. If we continue to get requests/input from other users that they'd like to see that change, I'll be open to it. Thanks for asking!